code/server/sv_ccmds.c

```diff
@@ -1181,70 +1181,71 @@ static int SV_Strlen( const char *str ) {
 SV_Status_f
 ================
 */
-static void SV_Status_f( void ) {
-	int			i, j, l;
-	client_t	*cl;
-	playerState_t	*ps;
-	const char		*s;
-	int			ping;
+static void SV_Status_f(void) {
 
-	// make sure server is running
-	if ( !com_sv_running->integer ) {
-		Com_Printf( "Server is not running.\n" );
+	int           i, j, l;
+	client_t      *cl;
+	playerState_t *ps;
+	const char    *s;
+	int           ping;
+	char          name[MAX_NAME_LENGTH];
+
+	if (!com_sv_running->integer) {
+		Com_Printf("Server is not running.\n");
 		return;
 	}
 
-	Com_Printf ("map: %s\n", sv_mapname->string );
+	Com_Printf("map: %s\n", sv_mapname->string);
+	Com_Printf("num score ping name            lastmsg address               qport rate \n");
+	Com_Printf("--- ----- ---- --------------- ------- --------------------- ----- -----\n");
 
-	Com_Printf ("cl score ping name            address                                 rate \n");
-	Com_Printf ("-- ----- ---- --------------- --------------------------------------- -----\n");
-	for (i=0,cl=svs.clients ; i < sv_maxclients->integer ; i++,cl++)
-	{
-		if (!cl->state)
+	for (i = 0, cl = svs.clients; i < sv_maxclients->integer; i++, cl++) {
+
+		if (!cl->state) {
 			continue;
-		Com_Printf ("%2i ", i);
-		ps = SV_GameClientNum( i );
-		Com_Printf ("%5i ", ps->persistant[PERS_SCORE]);
-
-		if (cl->state == CS_CONNECTED)
-			Com_Printf ("CON ");
-		else if (cl->state == CS_ZOMBIE)
-			Com_Printf ("ZMB ");
-		else
-		{
+		}
+
+		Com_Printf("%3i ", i);
+		ps = SV_GameClientNum(i);
+		Com_Printf("%5i ", ps->persistant[PERS_SCORE]);
+
+		if (cl->state == CS_CONNECTED) {
+			Com_Printf("CNCT ");
+		} else if (cl->state == CS_ZOMBIE) {
+			Com_Printf("ZMBI ");
+		} else {
 			ping = cl->ping < 9999 ? cl->ping : 9999;
-			Com_Printf ("%4i ", ping);
+			Com_Printf("%4i ", ping);
 		}
 
-		Com_Printf ("%s", cl->name);
-		
-		l = 16 - SV_Strlen(cl->name);
-		j = 0;
-		
-		do
-		{
-			Com_Printf (" ");
-			j++;
-		} while(j < l);
+		// Remove color codes not to break the padding.
+		Q_strncpyz(name, cl->name, sizeof(name));
```

Comment:
To be honest, I would prefer to completely remove the color codes from the player name in the `status` command. Indeed, many 3rd party tools relies on the output of this command. As far as I can tell, B3 (which is the most widely used administration software, I believe) won't complain about this change because it relies on a solid regular expression to parse each status command output line, but probably there are many tools out there that won't react well (I don't seem to find a reason why they should not be working anymore, but who knows).

I decided to simply strip the color codes, also because if someone wants to access the colored name of the player, `dumpuser` can be used.